mlflow/store/artifact/databricks_artifact_repo.py

```diff
@@ -90,12 +92,19 @@ def _complete_futures(futures_dict):
     """
     results = {}
     errors = {}
-    for future in as_completed(futures_dict):
-        key = futures_dict[future]
-        try:
-            results[key] = future.result()
-        except Exception as e:
-            errors[key] = repr(e)
+
+    with ArtifactProgressBar.chunks(
+        os.path.getsize(file),
+        f"Uploading file {file}",
```

Comment:
```suggestion
        f"Uploading {file}",
```

`file` here seems redudant to me.